File: java/2348.java
class Solution {
    public long zeroFilledSubarray(int[] nums) {
        int count = 0;
        long sac = 0;
        for (int i : nums)
            if (i == 0)
                sac += ++count;
            else
                count = 0;
        return sac;
    }
}

File: javascript/2348.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var zeroFilledSubarray = function (nums) {
    let count = 0;
    var sac = 0;
    for (let i of nums)
        if (i == 0)
            sac += ++count;
        else
            count = 0;
    return sac;
};
